### PR TITLE
support some ops, add test cases and remove duplicate source

### PR DESCRIPTION
**Pull request by fumihwh**
- SpaceToDepth
- Tile

### CODE REVIEW

**Comment by tjingrant** [comment]
good catch, must be a copy & past error on my side.

File: onnx_tf/backend.py [hunk 1]
```diff
@@ -1112,6 +1112,20 @@ def handle_softmax(cls, node, input_dict):
 
     return [tf.reshape(tf.nn.softmax(x - tf.reduce_max(x)), shape)]
 
+  @classmethod
+  def handle_space_to_depth(cls, node, input_dict):
+    x = input_dict[node.inputs[0]]
+    x_rank = len(x.get_shape())
+    support_cuda = cls.supports_device("CUDA")
+    storage_format, compute_format = cls.get_data_format(x_rank, support_cuda)
+    if support_cuda:
```
Comment (tjingrant) [inline onnx_tf/backend.py]: I know I did this in other places, but can you do something in the spirit of 
`storage_format != compute_format` 

just in case TF supports NCHW on CPU in the future.

Comment (tjingrant) [reply onnx_tf/backend.py]: Second thought, we really are just special-casing storage_format=NCHW and compute_format=NHWC. Maybe we should make this clear and I can work on it after this PR.

Comment (fumihwh) [reply onnx_tf/backend.py]: To first, even `storage_format != compute_format` (or sth like this) would be a little dirty. I will find a new way to deal with this.
Second, yes, and how is ONNX `NHWC`, data format thing going?

Comment (tjingrant) [reply onnx_tf/backend.py]: Still no response, I posted a new msg today asking for progress.
